dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java

```diff
@@ -634,7 +634,7 @@ public void testConfigChangeLoginTimeout320() throws Throwable {
 
         int loginTimeout;
         try {
-            loginTimeout = ds1.getLoginTimeout();
+            loginTimeout = ds2.getLoginTimeout();
```

Comment:
Why the switch to ds2?

Reply:
ds1 is a java.sql.DataSource type.  Not all database drivers support a loginTimeout property when using that type of DataSource.  Therefore, I switched to use ds2 which is a javax.sql.XADataSource type which should support loginTimeout on all database drivers.

Reply:
Thanks!